.github/workflows/staging_cicd.yml

```
@@ -0,0 +1,87 @@
+
+jobs:
+  ci:
+    name: CI
+    runs-on: ubuntu-latest
+    env:
+      SUBMODULE_TOKEN: ${{ secrets.SUBMODULE_ACTION_TOKEN }}
+      REGISTRY: ghcr.io
+      IMAGE_NAME: ${{ github.repository }} # <owner>/<repo>
+
+    outputs:
+      image: ${{ steps.set-image-output.outputs.image }}
```

actionlint (1.7.9)

property "set-image-output" is not defined in object type {}

(expression)